sdk/core/core-rest-pipeline/src/fetchHttpClient.ts

```diff
@@ -75,22 +75,30 @@ async function makeRequest(request: PipelineRequest): Promise<PipelineResponse>
 
   try {
     const headers = buildFetchHeaders(request.headers);
-    const requestBody = buildRequestBody(request);
+    const { streaming, body: requestBody } = buildRequestBody(request);
+    const requestInit: RequestInit = {
+      body: requestBody,
+      method: request.method,
+      headers: headers,
+      signal: abortController.signal,
+      credentials: request.withCredentials ? "include" : "same-origin",
+      cache: "no-store",
+    };
+
+    // According to https://fetch.spec.whatwg.org/#fetch-method,
+    // init.duplex must be set when body is a ReadableStream object.
+    // currently "half" is the only valid value.
+    if (streaming) {
+      (requestInit as any).duplex = "half";
```

Comment:
Looks like adding that property to the type is being tracked in https://github.com/microsoft/TypeScript-DOM-lib-generator/issues/1483